### PR TITLE
Ashley modal construction/styling

### PR DESCRIPTION
**Pull request by ashleyxnicolej**
- Created custom wireframe within Miro software for how the modal should display in a desktop browser vs. phone screen
- Ensured all content within the page is centered by adding a class to the body tag. 
- Added  <style> tags within the HTML head element to:
    -  center the “Let’s Get Started!” Button at the bottom of the Modal 
    - aid in animation process of the waving hand emoji in the Modal
- Constructed + Implemented Modal:
    - placed the modal below the <body> tag to ensure all content is loaded on the webpage prior to modal display
- Activated the Modal 
- Added Custom Content for Modal 
    - included waving hand emoji/ custom animation 
- Created a custom <div> class for the modal footer to sync with the added <style> tag in the HTML head element 

-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,43 @@
...
+  				30% { transform: rotate(14.0deg) }
+   				40% { transform: rotate(-4.0deg) }
+  				50% { transform: rotate(10.0deg) }
+  				60% { transform: rotate( 0.0deg) }
+		
+			}
+		</style>
 	</head>
 
-	<body>
```
Comment (KJUDALL) [inline index.html:10]: Good class addition here! I'm curious as to why my page appears centered without having the class in my html code, but I think I'll understand better when Casey adds his code in.

File: index.html [hunk 2]
```diff
@@ -53,4 +87,43 @@ <h1 class="title is-1 has-text-centered is-spaced has-text-primary-30">Grocery T
 
 	</body>
 
-</html>
```
Comment (KJUDALL) [inline index.html:56]: This looks awesome, nice job. I'm assuming you had to put it at the bottom of the HTML to let the rest of the page load first?

**Review by KJUDALL — APPROVED** [review]
Left 2 comments, but looks really good. Excited to see the modal in action once everything is merged in.

File: index.html (hunk 1 above)
Comment (ashleyxnicolej) [reply index.html:10]: Thanks! I also noticed the default content on the web page was centered, however when I ran my syntax I spotted where the content in the ul wasn't in the center. I also centered the modal + content within the modal as well (which was only seen on my end bc I hadn't merged yet)

File: index.html (hunk 2 above)
Comment (ashleyxnicolej) [reply index.html:56]: That is correct. The Modal had to be added outside of the body element and at the bottom of the HTML document in order for the rest of the content on the webpage to load properly without the Modal destructing the layout.

**Pull request merged**